<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPageSetting"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="actionFind"/>
    <addaction name="actionGoto"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionData"/>
    <addaction name="separator"/>
    <addaction name="actionFontSetting"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>查看</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>缩放</string>
     </property>
     <addaction name="actionReduce"/>
     <addaction name="actionEnlarge"/>
     <addaction name="actionToDefault"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="actionShowStatusBar"/>
    <addaction name="actionAutoChangeLine"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionPageSetting">
   <property name="text">
    <string>页面设置</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="text">
    <string>转到</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actionData">
   <property name="text">
    <string>时间/日期</string>
   </property>
  </action>
  <action name="actionFontSetting">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="actionShowStatusBar">
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="actionAutoChangeLine">
   <property name="text">
    <string>自动换行</string>
   </property>
  </action>
  <action name="actionReduce">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionEnlarge">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actionToDefault">
   <property name="text">
    <string>还原默认比例</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
